libs/components/toast/src/lib/modules/toast/toast.component.html

```diff
@@ -5,16 +5,19 @@
   <div
     class="sky-toast sky-shadow sky-rounded-corners sky-box sky-elevation-24"
     [ngClass]="classNames"
-    [attr.aria-live]="ariaLive"
-    [attr.role]="ariaRole"
   >
     <div aria-hidden="true" class="sky-toast-icon-theme-default">
       <sky-icon [icon]="icon" />
     </div>
     <div aria-hidden="true" class="sky-toast-icon-theme-modern">
       <sky-icon-stack [baseIcon]="baseIcon" [topIcon]="topIcon" />
     </div>
-    <div #toastContent class="sky-toast-content" skyId="toastContent">
+    <div
+      #toastContent
+      class="sky-toast-content"
+      skyId="toastContent"
+      role="alert"
```

Comment:
The `alert` role has special handling that reads it out even on initial rendering. In speaking with @Blackbaud-MattGregg we were comfortable with just making all toasts use the `alert` role.

The previous implementation was also on the entire toast which caused the close button to read out when it should not have. The previous `aria-live` implementation is redundant as the `alert` role automatically makes the element a live region.